packages/venia-ui/lib/components/CheckoutPage/AddressBook/addressBook.css

```diff
@@ -40,16 +49,24 @@
 }
 
 .addButton {
-    border: 1px dashed rgb(var(--venia-global-color-border));
-    border-radius: 5px;
+    border: 2px dashed rgb(var(--venia-global-color-gray-400));
+    border-radius: 0.375rem;
     font-size: 0.875rem;
     font-weight: 600;
     transition: border-color 384ms var(--venia-global-anim-standard);
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    color: rgb(var(--venia-brand-color-1-700));
+}
+.addText {
+    margin-left: 0.625rem;
 }
 
+.addButton:focus,
```

Comment:
I think focus treatment should be the same as normal buttons (the shadow).

Reply:
Changed to be the same as other buttons.